IfcElectricFlowStorageDevice Rail Import BIMQ 2021-07-21T07:39:34.386 (#186)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/o/OutputCurrent_3hgdbiYpLERhPeEQfTel7i/Documentation.md

> The actual current and operable range as input of the equipment.

Author: ```suggestion
The actual current and operable range output from the equipment.
```
